scipy/fftpack/tests/test_basic.py

```diff
@@ -324,6 +324,29 @@ def test_invalid_sizes(self):
         assert_raises(ValueError, rfft, [])
         assert_raises(ValueError, rfft, [[1,1],[2,2]], -5)
 
+    # See Issue #5790
```

Comment:
CMIW, but I've never really seen or heard of any sort of convention in terms of referring to GitHub issues in this code-base or its close cousin `numpy` for that matter.  Also, I think the test naming and comments I put should be sufficient in itself to indicate what is being tested.